### DISCUSSION TITLE
Support for version 3.0

### DISCUSSION THREAD

**Discussion by sagelga** [Andrewthe13th/Inventory_Kamera#359] [Ideas or Feature requests]
With the upcoming version of 3.0, I would like to make this project supports the version 3.0 since day one. 

If it possible we can use the leaked content from beta testers and adding it to the scanner, available for use when the time comes. 

I understand that we seems not to do that (or shouldn’t do that), but darn, this is the 3rd major release of Genshin Impact. There should be more users online during this event. 

If anyone got some ideas or information about how we can do that, comment below!

**Comment by Cupcak3** [comment]
Kamera has code to handle Dendro element when it releases. The auto updater additionally checks for dimbreath repo updates for data on new characters and (relevant) items with each dump of the game. Unless there is a UI update to the character and inventory screens, it should handle whatever comes with 3.0.
